compose-builder/add-security.yml

```diff
@@ -239,17 +238,18 @@ services:
       - common-sec-stage-gate.env
     environment:
       KONGURL_SERVER: kong
-      ROUTES_EDGEX-CORE-DATA_HOST: edgex-core-data
-      ROUTES_EDGEX-CORE-METADATA_HOST: edgex-core-metadata
-      ROUTES_EDGEX-CORE-COMMAND_HOST: edgex-core-command
-      ROUTES_EDGEX-SUPPORT-NOTIFICATIONS_HOST: edgex-support-notifications
-      ROUTES_EDGEX-SUPPORT-SCHEDULER_HOST: edgex-support-scheduler
-      ROUTES_RULES-ENGINE_HOST: edgex-kuiper
-      ROUTES_DEVICE-VIRTUAL_HOST: edgex-device-virtual
+      ROUTES_CORE_CONSUL_HOST: edgex-core-consul
+      ROUTES_CORE_DATA_HOST: edgex-core-data
+      ROUTES_CORE_METADATA_HOST: edgex-core-metadata
+      ROUTES_CORE_COMMAND_HOST: edgex-core-command
+      ROUTES_SUPPORT_NOTIFICATIONS_HOST: edgex-support-notifications
+      ROUTES_SUPPORT_SCHEDULER_HOST: edgex-support-scheduler
+      ROUTES_RULES_ENGINE_HOST: edgex-kuiper
+      ROUTES_DEVICE_VIRTUAL_HOST: device-virtual
       ROUTES_EDGEX-CORE-CONSUL_HOST: edgex-core-consul
```

Comment:
this one is duplicates or redundant. remove the duplicate

Reply:
Yep, good catch